src/main/java/com/dev/cinema/model/dto/TicketResponseDto.java

```diff
@@ -0,0 +1,10 @@
+package com.dev.cinema.model.dto;
+
+import lombok.Data;
+
+@Data
+public class TicketResponseDto {
```

Comment:
I think we can remove this Dto and use `List<Long> ticketIds` field in ShoppingCartResponseDto and OrderResponseDto instead